# The following are the proportions of girl births in Vienna for each month in 1908 and 1909 (out of an average of 3900 births per month):

In [1]:
girls.1908 <- c(.4777, .4875, .4859, .4754, .4874, .4864, .4813, .4787, .4895, .4797, .4876, .4859)
girls.1909 <- c(.4857, .4907, .5010, .4903, .4860, .4911, .4871, .4725, .4822, .4870, .4823, .4973)

In [2]:
n <- 3900

In [3]:
girls <- c(girls.1908, girls.1909)

*Compute the standard deviation of these proportions and compare to the standard deviation that would be expected if the sexes of babies were inde- pendently decided with a constant probability over the 24-month period.*

In [4]:
std.obs <- sd(girls)

In [5]:
std.obs

[1] 0.006409724

In [6]:
p.obs <- mean(girls)

In [7]:
p.obs

[1] 0.485675

In [8]:
stderror.expected <- sqrt(p.obs*(1-p.obs)/n)

In [9]:
stderror.expected

[1] 0.008003121

*The actual and theoretical standard deviations from (a) differ, of course. Is this difference statistically significant? (Hint: under the randomness model, the actual variance should have a distribution with expected value equal to the theoretical variance, and proportional to a χ2 with 23 degrees of freedom.)*

Relies on:
https://raw.githubusercontent.com/Fossj117/gelman_hill/master/Ch2/Ch2_ex.md
http://www.milefoot.com/math/stat/ci-variances.htm

In [10]:
df <- 23
alpha <- 0.05

In [11]:
chisq.upper <- qchisq(alpha/2., df)

In [12]:
sd_upper <- sqrt(df*(std.obs^2)/chisq.upper)

In [13]:
chisq.lower <- qchisq(1 - (alpha/2.), df)

In [14]:
sd_lower <- sqrt(df*(std.obs^2)/chisq.lower)

In [15]:
sd_lower

[1] 0.004981725

In [16]:
sd_upper

[1] 0.008991309

As the observed standard deviation of 0.0064 is in the range [0.005, 0.009], we conclude that the difference is *not signficant* at 5%. 